dbms/tests/users.d/multiprofile.xml

```diff
@@ -0,0 +1,29 @@
+<yandex>
+  <profiles>
+    <profile_1>
+      <max_parallel_replicas>2</max_parallel_replicas>
+    </profile_1>
+    <profile_2>
+      <max_network_bytes>1234567890</max_network_bytes>
+    </profile_2>
+    <profile_3>
+      <max_insert_block_size>654321</max_insert_block_size>
```

Comment:
The settings constraints are also defined in profiles, so I suppose we should collect constraints from all inherited profiles.

Reply:
Done in the latest commits